### PR TITLE
chore(protocol): update submodules

### PR DESCRIPTION
**Pull request by dantaik**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,16 +1,25 @@
...
+	url = https://github.com/Vectorized/solady
+	tag = v0.0.166
+[submodule "packages/protocol/lib/p256-verifier"]
+	path = packages/protocol/lib/p256-verifier
+	url = https://github.com/taikoxyz/p256-verifier
+	commit = c8210cd99f9f81a58e25a6a8afdb992fefbd9748
+[submodule "packages/protocol/lib/automata-dcap-v3-attestation"]
+	path = packages/protocol/lib/automata-dcap-v3-attestation
+	url = https://github.com/automata-network/automata-dcap-v3-attestation
+	commit = 80eba1c014c193d4e83e07ac7d015cea8ebf1232
```
Comment (adaki2004) [inline .gitmodules:25]: @smtmfft is this the one we are using on our branch , right ?

Comment (dantaik) [reply .gitmodules:25]: this is the latest commit, they don't have any release.

Comment (dantaik) [reply .gitmodules:25]: All submodules should specify something (commit, tag, branch) so the code we use is deterministic.

Comment (adaki2004) [reply .gitmodules:25]: I know, but we need to compare this is the one we indeed using, because i brought over this from @smtmfft repo.

Comment (dantaik) [reply .gitmodules:25]: Then we need to clone @smtmfft's repo, then use that github repo's URL as the submodule URL.

Comment (dantaik) [reply .gitmodules:25]: Feel free to change the config in the .gitmodules file.

File: .gitmodules [hunk 2]
```diff
@@ -1,16 +1,25 @@
...
+	tag = v4.8.2
+[submodule "packages/protocol/lib/solady"]
+	path = packages/protocol/lib/solady
+	url = https://github.com/Vectorized/solady
+	tag = v0.0.166
+[submodule "packages/protocol/lib/p256-verifier"]
+	path = packages/protocol/lib/p256-verifier
+	url = https://github.com/taikoxyz/p256-verifier
+	commit = c8210cd99f9f81a58e25a6a8afdb992fefbd9748
+[submodule "packages/protocol/lib/automata-dcap-v3-attestation"]
```
Comment (dantaik) [inline .gitmodules:22]: TODO: dani. please change this to the right URL.

File: packages/protocol/README.md [hunk 3]
```diff
@@ -4,7 +4,33 @@ This package contains rollup contracts on both L1 and L2, along with other assis
...
+git submodule update --recursive --remote
+
+```
+
+If some git submodules are not initialized correctly, use the following commands in the root directory:
+
+```sh
+git submodule add  \
+--name packages/protocol/lib/automata-dcap-v3-attestation\
+https://github.com/automata-network/automata-dcap-v3-attestation \
```
Comment (dantaik) [inline packages/protocol/README.md:19]: please also change the URL here.

**Comment by adaki2004** [comment]
Closing as implemented these changes:
- adding smtmfft's fork into taiko organization, then reference as submodule
- update readme.md
on branch [onchain_remote_attestation](https://github.com/taikoxyz/taiko-mono/tree/onchain_remote_attestation) with the latest 2 commits there ([99d7a60](https://github.com/taikoxyz/taiko-mono/pull/15559/commits/99d7a60ab716f86483f39f74df5b777d77296f69) and [4d4ebad](https://github.com/taikoxyz/taiko-mono/pull/15559/commits/4d4ebadb64cf493e33bf5bcf9151bc60ba081e05))

**Pull request closed without merging**